### PR TITLE
Implemented message display in UI when no picklists found (#31fnaw8)

### LINKED ISSUE #71: Display a message when no picklist found
## Current behavior

If there are on picklist found then a toast is displayed and screen is empty

## Expected behavior

There should be a permanent message `There are no picklists available`

## Can you handle fixing this bug by yourself?

- [ ] YES
- [ ] NO

## Environment details

- Browser: 
- OS: 
- Code Version:

### PR DESCRIPTION
**Pull request by k2maan**
### Related Issues

Closes #71 

### Short Description and Why It's Useful

Added message and toast when no picklists are there.

### Screenshots of Visual Changes before/after (If There Are Any)

Picking - HotWax Commerce

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/store/modules/picklist/actions.ts [hunk 1]
```diff
@@ -19,6 +19,8 @@ const actions: ActionTree<PicklistState, RootState> = {
       if (resp.status === 200 && resp.data.pickingList && !hasError(resp)) {
         commit(types.PICKLISTS_UPDATED, { list: resp.data.pickingList })
         return resp.data;
+      } else if(!resp.data.pickingList) {
+        showToast(translate('No picklist found'));
```
Comment (ymaheshwari1) [inline src/store/modules/picklist/actions.ts:23]: We will get `No picklist found` message even when api fails.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/store/modules/picklist/actions.ts (hunk 1 above)
Comment (k2maan) [reply src/store/modules/picklist/actions.ts:23]: Added the check sir

File: src/store/modules/picklist/actions.ts [hunk 2]
```diff
@@ -16,12 +16,11 @@ const actions: ActionTree<PicklistState, RootState> = {
     try {
       resp = await PicklistService.getPicklists();
 
-      if (resp.status === 200 && resp.data.pickingList && !hasError(resp)) {
+      if(resp.status === 200 && resp.data.pickingList && !hasError(resp)) {
```
Comment (ymaheshwari1) [inline src/store/modules/picklist/actions.ts:19]: Remove unwanted change.

File: src/store/modules/picklist/actions.ts [hunk 3]
```diff
@@ -16,12 +16,11 @@ const actions: ActionTree<PicklistState, RootState> = {
...
         commit(types.PICKLISTS_UPDATED, { list: resp.data.pickingList })
         return resp.data;
       } else {
-        showToast(translate('Something went wrong'));
-        console.error("error", resp.data._ERROR_MESSAGE_);
```
Comment (ymaheshwari1) [inline src/store/modules/picklist/actions.ts:24]: Revert console.error change.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

**Review by ymaheshwari1 — APPROVED** [review]

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**